CHANGELOG.rst

```diff
@@ -1,6 +1,24 @@
 Changelog
 =========
 
+1.13.0 (2022-12-16)
+-------------------
+
+Feature
+"""""""
+
+- Support assume role for AWS base async hook
```

Comment:
Please write this from a usage perspective

Reply:
Fixed 
- Fix ``AwsBaseHookAsync`` to support assume role ``role_arn`` passed via ``extra_config`` in the airflow connection.

Reply:
It is an enhancement rather than a fix.

Enhance ``AwsBaseHookAsync`` to support assume role role_arn passed via extra_config in the airflow connection.